### PR TITLE
`yii\bootstrap\Html` helper added

### LINKED ISSUE #42: Icons support
Migrated from https://github.com/yiisoft/yii2/issues/5207

### PR DESCRIPTION
**Pull request by klimov-paul**
Fixes #42 and #43.

`yii\bootstrap\Html` helper added.

### CODE REVIEW

**Comment by samdark** [comment]
Nice idea.

**Comment by klimov-paul** [comment]
PR updated.
@yiisoft/core-developers please provide your feedback.

File: ActiveForm.php [hunk 1]
```diff
@@ -98,4 +97,13 @@ public function init()
         }
         parent::init();
     }
+
+    /**
+     * @inheritdoc
+     * @return ActiveField the created ActiveField object
+     */
+    public function field($model, $attribute, $options = [])
```
Comment (samdark) [inline ActiveForm.php]: Isn't it safe to remove it? It does nothing other than calling parent implementation.

File: ActiveField.php [hunk 2]
```diff
@@ -291,6 +290,20 @@ public function radioList($items, $options = [])
     }
 
     /**
+     * Renders Bootstrap static form control.
+     * @param array $options the tag options in terms of name-value pairs. These will be rendered as
+     * the attributes of the resulting tag. There are also a special options:
+     * - encode: boolean, whether value should be HTML-encoded or not.
+     * @return $this the field object itself
+     */
+    public function staticControl($options = [])
```
Comment (samdark) [inline ActiveField.php]: What's the use case for it?

File: ActiveForm.php (hunk 1 above)
Comment (klimov-paul) [inline ActiveForm.php]: It is added for IDE type-hint support.

Comment (samdark) [inline ActiveForm.php]: It could be done via `@method` annotation w/o actual implementation.

Comment (klimov-paul) [inline ActiveForm.php]: Here I am using same approach as:
https://github.com/yiisoft/yii2/blob/master/framework/db/ActiveQuery.php#L128

Comment (samdark) [inline ActiveForm.php]: OK.

**Comment by samdark** [comment]
Overall, it looks fine.

File: ActiveField.php [hunk 3]
```diff
@@ -291,6 +290,20 @@ public function radioList($items, $options = [])
     }
 
     /**
+     * Renders Bootstrap static form control.
+     * @param array $options the tag options in terms of name-value pairs. These will be rendered as
+     * the attributes of the resulting tag. There are also a special options:
+     * - encode: boolean, whether value should be HTML-encoded or not.
```
Comment (cebe) [inline ActiveField.php]: a newline is required before this list, otherwise it would not render properly.

**Comment by cebe** [comment]
except the phpdoc formatting which needs to be adjusted for all methods in BaseHtml too, it looks fine to me.

**Pull request merged**